qemu/tests/cfg/throttle_operation_test.cfg

```diff
@@ -0,0 +1,91 @@
+- throttle_operation_test:
+    type = throttle_operation_test
+    qemu_force_use_drive_expression = no
```

Comment:
```suggestion
```
This default of `qemu_force_use_drive_expression` is `no` in the avocado-vt framework.